packages/squiggle-lang/__tests__/TS/Score_test.ts

```diff
@@ -0,0 +1,19 @@
+import { testRun } from "./TestHelpers";
+
+describe("KL divergence", () => {
+  test.skip("by integral solver agrees with analytical", () => {
```

Comment:
why is this also skipped? Maybe delete and re-add in the next story?

Reply:
I agree.